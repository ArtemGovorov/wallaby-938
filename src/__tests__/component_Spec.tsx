import { TestComponent } from "../component";
import * as React from "react";
import { create } from "react-test-renderer";

describe("component", () => {
    it("Should be rendered correctly", () => {
        const tree = create(<TestComponent a="test string" b={5} />).toJSON();
        expect(tree).toMatchSnapshot();
        expect(tree).toMatchSnapshot();
    });
    
    it("Should be rendered correctly with different props", () => {
        const tree = create(<TestComponent a="another string" b={6} />).toJSON();
        expect(tree).toMatchSnapshot();
        expect(tree).toMatchSnapshot();
    });
});