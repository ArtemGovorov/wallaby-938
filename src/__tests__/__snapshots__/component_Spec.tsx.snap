exports[`component Should be rendered correctly 1`] = `
<div
  className="root">
  <div
    className="string">
    String is 
    test string
  </div>
  <div
    className="number">
    Number is 
    5
  </div>
</div>
`;

exports[`component Should be rendered correctly 2`] = `
<div
  className="root">
  <div
    className="string">
    String is 
    test string
  </div>
  <div
    className="number">
    Number is 
    5
  </div>
</div>
`;

exports[`component Should be rendered correctly with different props 1`] = `
<div
  className="root">
  <div
    className="string">
    String is 
    another string
  </div>
  <div
    className="number">
    Number is 
    6
  </div>
</div>
`;

exports[`component Should be rendered correctly with different props 2`] = `
<div
  className="root">
  <div
    className="string">
    String is 
    another string
  </div>
  <div
    className="number">
    Number is 
    6
  </div>
</div>
`;
